### DISCUSSION TITLE
Welcome to SSH Discussions!

### DISCUSSION THREAD

**Discussion by melvillewong** [melvillewong/SSH#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by melvillewong** [comment]
Hi everyone, I'm Melville, and I'm passionate about developing efficient and creative solutions. I'm excited to contribute to implementing a new smart scheduling feature for SSH and to collaborate with the community to build something impactful!

Nice to meet you all!

**Comment by chapwiz** [comment]
Hi, my name is Chapman. Looking forward to contributing to such exciting project, with collaboration in place. It's my pleasure to meet you guys.
